content/source/docs/plugin/framework/publishing.html.md

```diff
@@ -15,21 +15,35 @@ version 5 of the Terraform protocol, but providers built using the framework are
 
 ## Add a Version Manifest
 
-When you upload your new provider version to GitHub:
-
-1. Next to the zip files containing your binaries for each platform, include a file named `terraform-provider-$NAME_$VERSION_manifest.json` where `$NAME` is your provider's name (e.g. `random`) and `$VERSION` is your provider's version (e.g. `1.2.3`).
+Add the following contents to a file (e.g. `terraform-registry-manifest.json`). The version number in the file is the version of the manifest format, not the version of your provider.
```

Comment:
I want to push back here on this rearrange only because like this users might name the file the wrong thing which is annoying. I wonder if we can move the information about how to name the file back up to be with this content and then change the below to:

When you upload your new provider version to GitHub:
1. Add the manifest file next to the zip files containing your binaries for each platform.
2. Include the SHA-256 checksum of this JSON file in your `SHA256SUMS` file.

Reply:
I think this may be confusing for a few reasons:

* A lot of "versioned" file naming conventions for publishing are outlined in the [Manually Preparing a Release section of the Publishing Providers page](https://www.terraform.io/docs/registry/providers/publishing.html#manually-preparing-a-release) while this page is currently separate from those details because the manifest file is _technically_ only required for publishing providers using protocol version 6. It should, however, work fine for publishing existing providers as well, and it will be a recommendation we are going to document so developers will have less of a need to discover the existence of the manifest file.
* We have updated https://github.com/hashicorp/terraform-provider-scaffolding to include the manifest file statically named as `terraform-registry-manifest.json` and updated that `.goreleaser.yml` to include this suggested configuration.
* We wouldn't expect developers to continually keep updating the manifest file name prior to every time they cut a release just to match the expected release naming. 😅 

All that being said, I'm happy to adjust this as appropriate. There is the caveat that this page can likely now be removed and redirected to the Publishing Providers page by adding some of manifest file information and appropriately suggesting when to use protocol version 5 versus 6 in that file. I figured that change was worth more discussion though, which is why I separated these changes out first. 👍

Reply:
ooooo! okay! This makes sense to me. Then I wonder the reverse haha - could we take out "Next to the zip files containing your binaries for each platform, include a file named `terraform-provider-$NAME_$VERSION_manifest.json` where `$NAME` is your provider's name (e.g. `random`) and `$VERSION` is your provider's version (e.g. `1.2.3`)." and replace it with "Add the manifest file next to the zip files containing your binaries for each platform."  

I think I just don't want to explain the naming twice, if that makes sense. But now that you've explained it - I think we leave what you wrote up top and delete the extraneous explanation below.  Let me know if I'm still missing something! and okay yes - from what you're saying it does make sense to combine this in with the rest of the publishing info and just point users there.  I'd be happy to help with/look at a separate PR that does that in the next few weeks!